### ISSUE TITLE
Dark theme: code completion selection does not have enough contrast -> align with other popular dark themes

### ISSUE THREAD

**Issue by thomasritter** [eclipse-platform/eclipse.platform.ui#1688]
Yesterday, I had an interesting call with a user of our source based editor. The user uses Eclipse on a large monitor with the Eclipse dark theme. He showed me that the contrast of the code completion is not high enough for him. Therefore, he always:

a) Opens the code completion
b) Uses tab to highlight the selection. With the highlight the contrast is much better

Here the comparison between:

Selected via tab:
<img width="427" alt="image" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/41768/e2b08145-cc17-4193-a9e6-62452a0d5545">

Without tab selection:
<img width="381" alt="image" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/41768/e9acc405-06c3-450b-82fb-78186078d5b5">

Now, imagine the first selection on a large monitor with a high resolution. The contrast is simply not enough.

Here some examples how other IDEs render the selection without the need to press tab.

Intellij:
<img width="336" alt="Screenshot 2024-02-15 at 09 02 53" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/41768/bcdb509d-dcfb-4dee-b976-64da088ca439">

VS Code:
<img width="684" alt="Screenshot 2024-02-15 at 09 02 41" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/41768/cd297995-ec76-4326-9f0c-a948bb54c9ab">

Both use a light blue which provides more contrast.

I would propose that the dark theme in Eclipse gets adjusted, too. Having to use tab feels like a hack/workaround. What do you think? @BeckerWdf 

- [X ] I verified [latest Integration Build of Eclipse SDK](https://download.eclipse.org/eclipse/downloads/) doesn't provide the feature.

**Comment by HeikoKlare** [comment]
I agree that having better highlighting would be great. Note that on Windows the issue is even worse, as highlighting in dark theme is _always_ hardly visible: https://github.com/eclipse-platform/eclipse.platform.swt/issues/811

Also note that the GTK implementation already does what you expect. Highlighting of the selected element is always done in a well visible way (no matter whether using tab to have focus or not):
![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/755472/582395ef-ffb7-4042-804a-258a2625a905)

I would really appreciate an improvement here (in particualar if it also applies to Windows), I am only a little concerned that this might be something that the native widgets do not allow to style (at least on Windows).

Edit: Just saw that it actually seems to be possible to modify: https://github.com/eclipse-platform/eclipse.platform.ui/pull/1690

**Comment by Christopher-Hermann** [comment]
With some tweaking of #1690 I successfully activated the new selection highlight coloring feature for code completion. This would resolve the corresponding issue across all platforms.
![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/79514265/27a7cffe-5869-4c71-8446-855eda0157af)

Another fix (at least for MacOS) would be to set the focus to the code completion every time it is opened. This would eliminate the extraneous step of having to manually select the focus with the tab key.

I hope I found some time the next days/weeks to look into this issue and also finish the work on the mentioned PR #1690.

**Comment by Christopher-Hermann** [comment]
Fixed with the latest commit in the PR. Code completion proposal selection is now always drawn in the selection color.
![image](https://github.com/eclipse-platform/eclipse.platform.ui/assets/79514265/52460d1b-75ac-4a49-a836-4d3703c8c08b)
